File: src/18장/18-6.py
import sys
n = int(input())
count_sort = [0] * 10001
for i in range(n):
    count_sort[int(sys.stdin.readline())] += 1
for i in range(10001):
    if count_sort[i] != 0:
        for j in range(count_sort[i]):
            print(i)

File: src/18장/18-6.java
import java.util.*;
import java.io.*;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
		StringBuilder sb = new StringBuilder();
		int N = Integer.parseInt(br.readLine());
		int[] count_sort = new int[N];
		for(int i = 0; i < N; i++){
			count_sort[i] = Integer.parseInt(br.readLine());
		}
		Arrays.sort(count_sort);
		for(int i = 0; i < N; i++){
			sb.append(count_sort[i]).append('\n');
		}
		System.out.println(sb);
	}
}